lib/json_logger.rb

```diff
@@ -1,3 +1,6 @@
+# Don't merge the datadog trace info in some environments as it clutters up logs
+DATADOG_TRACE_INFO_ENABLED = !(Rails.env.development? || Rails.env.test?)
```

Comment:
the trace info in dev can be really noisy, so setting it here as a constant to turn it off but making it easy to toggle back on. Sometimes the trace info is useful if you are debugging how the datadog tracing pieces are working.